modern/src/components/registration/LoginForm.js

```diff
@@ -57,6 +60,11 @@ const LoginForm = () => {
     }
   }
 
+  const handleLanguageChange = e => {
+    setLanguage(e.target.value);
+    languageContext.setLanguage(e.target.value);
```

Comment:
We should be able to use single source of truth.

Reply:
Now it only uses the context